modules/node-shared/src/main/scala/org/tessellation/node/shared/http/routes/SnapshotRoutes.scala

```diff
@@ -40,78 +43,102 @@ final case class SnapshotRoutes[F[_]: Async: Hasher: KryoSerializer, S <: Snapsh
   implicit def binaryAndJsonEncoders[A <: AnyRef: Encoder]: List[EntityEncoder[F, A]] =
     List(BinaryCodec.encoder[F, A], CirceEntityEncoder.circeEntityEncoder[F, A])
 
+  private def badRequestNodeNotReady(currentState: NodeState): F[Response[F]] =
+    BadRequest(("message" ->> s"Node should be in READY state before fetching. Current state: $currentState") :: HNil)
```

Comment:
The enum NodeState specifies state as `Ready` not `READY`

Reply:
Updated